### PR TITLE
New try for rils-rols==1.5.15

### PR DESCRIPTION
**Pull request by kartelj**
I have removed certain os.environment settings from setup.py. Maybe these settings affected previous ITEA fail.

### CODE REVIEW

**Comment by folivetti** [comment]
sabotage!! 😂

Setting values to `os.environment` may be a reason to break itea, tir, chiefly as I set the path to the external libs I'm using. Though I would assume these CIs are run independently.

We need to figure it out a way that everybody can set their envs without interfering with each other.

**Comment by kartelj** [comment]
Hi Fabricio,

I accept the guilt :)
However, it seems that this is not the problem :(
I have removed os.environment settings from setup.py now, but ITEA, TIR and GPGOMEA still fail. 
https://github.com/kartelj/rils-rols/blob/master/setup.py

Does it fail without RILS-ROLS as well?

**Comment by folivetti** [comment]
looking at the history, it seems it was failing before rils-rols, so I think it's not your fault :-)

I´ ll have a look at that later!

File: algorithms/rils-rols/regressor.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+eval_kwargs = {'scale_x': False, 'scale_y': False}
+# This is a setup for ground-truth instances (feynman and strogatz), for black-box just change max fit calls to 500k, i.e., max_fit_calls=500*1000,
+# if there is still that requirement (as in paper), otherwise keep 1 million. 
+est = RILSROLSRegressor(max_seconds=2*60*60, max_fit_calls=1000*1000, max_complexity=50, sample_size=0, verbose=True)
+
+def model(est):
+    return str(est.model_string())
+
+
+def complexity(est):
```
Comment (lacava) [inline algorithms/rils-rols/regressor.py:14]: don't need this function any more

**Review by lacava — CHANGES_REQUESTED** [review]

**Comment by lacava** [comment]
> Setting values to os.environment may be a reason to break itea, tir, chiefly as I set the path to the external libs I'm using. Though I would assume these CIs are run independently.

Yea, this shouldn't matter for ITEA/TIR, they are in a separate python environment

File: algorithms/rils-rols/regressor.py (hunk 1 above)
Comment (kartelj) [reply algorithms/rils-rols/regressor.py:14]: Sure, shall I PR or just remove it directly?
You can delete it as well if you want.

Comment (lacava) [reply algorithms/rils-rols/regressor.py:14]: If you update your pr branch it should show up here.  please go ahead

**Comment by lacava** [comment]
this looks good. once it finishes rebase and passes tests we should be good to merge. thanks for contributing a method!

**Review by lacava — APPROVED** [review]

**Pull request merged**